provider/ovh/ovh.go

```diff
@@ -0,0 +1,66 @@
+package ovh
+
+import (
+	"fmt"
+
+	"github.com/ovh/go-ovh/ovh"
+	"github.com/messagebird/sachet"
+)
+
+type Config struct {
+	Endpoint             string `yaml:"endpoint"`
+	ApplicationKey       string `yaml:"application_key"`
+	ApplicationSecret    string `yaml:"application_secret"`
+	ConsumerKey          string `yaml:"consumer_key"`
+
+	ServiceName          string `yaml:"service_name"`
+	Sender               string `yaml:"sender"`
+}
+
+type Ovh struct {
+	client *ovh.Client
+	config *Config
+}
+
+func NewOvh(config Config) (*Ovh, error) {
+	client, err := ovh.NewClient(
+		config.Endpoint,
+		config.ApplicationKey,
+		config.ApplicationSecret,
+		config.ConsumerKey,
+	)
+	if err != nil {
+		return nil, err
+	}
+	return &Ovh{
+		client: client,
+		config: &config,
+	}, nil
+}
+
+func (ovh *Ovh) Send(message sachet.Message) error {
+	var err error = nil
+	switch message.Type {
+	case "", "text":
+		type ovhSMS map[string]interface{}
+		sms := make(ovhSMS)
+		sms["message"] = message.Text
+		sms["noStopClause"] = false
+		if &ovh.config.Sender != nil {
```

Comment:
You can get the "sender" from the "Originator" on the Message.
https://github.com/messagebird/sachet/blob/master/sachet.go#L9
you shouldn't add it to the configuration.